packages/rancher-gatekeeper/package.yaml

```diff
@@ -1,4 +1,4 @@
-url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-v3.1.0.tgz
+url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.1.1.tgz
```

Comment:
You don't actually need to remove the old tgz's because you are creating a new version. You can leave for upgrade testing, or remove them to clean up in general. Either way works.

Reply:
I removed them for cleanup since I would consider this version the best starting point for being a GA and cleaner.